packages/block-library/src/buttons/style.scss

```diff
@@ -12,4 +12,8 @@
 
 .wp-block-buttons.aligncenter {
 	text-align: center;
+
+	.wp-block-button:last-child {
+		margin-right: 0;
+	}
```

Comment:
good catch. Do you think we should remove this margin in all cases, not just the aligned centered buttons? (And the left margin in case of the aligned right ones?

Reply:
Makes sense. Done in the last commit: aed2ea722e7cf7e3c09638f1222650c14de30397.

| Before | After |
| --- | --- |
| ![imatge](https://user-images.githubusercontent.com/3616980/85204321-fdadd580-b313-11ea-9319-fdf2abbc5b83.png) | ![imatge](https://user-images.githubusercontent.com/3616980/85204311-f2f34080-b313-11ea-8019-5061c50729b7.png) |